### DISCUSSION TITLE
[BUG] Guiscrcpy wont show android device

### DISCUSSION THREAD

**Discussion by Komarams** [srevinsaju/guiscrcpy#397] [General]
**Describe the bug**
I installed guiscrcpy with flatpak . On running i get the gui, when I connect my android device (nougat 7.1) and click "Start scrcpy" button , nothing happens only swipe panel opens up . I get an error in console as 

`Qt: Session management error: Could not open network socket
qt.glx: qglx_findConfig: Failed to finding matching FBConfig for QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize 0, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
No XVisualInfo for format QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize 0, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
Falling back to using screens root_visual.
qt.glx: qglx_findConfig: Failed to finding matching FBConfig for QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize -1, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
No XVisualInfo for format QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize -1, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
Falling back to using screens root_visual.
qt.glx: qglx_findConfig: Failed to finding matching FBConfig for QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize 0, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
No XVisualInfo for format QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 1, greenBufferSize 1, blueBufferSize 1, alphaBufferSize 0, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::SingleBuffer, swapInterval 1, colorSpace QSurfaceFormat::DefaultColorSpace, profile  QSurfaceFormat::NoProfile)
Falling back to using screens root_visual.
scrcpy 1.24 <https://github.com/Genymobile/scrcpy>
/app/share/scrcpy/scrcpy-server: 1 fil...ed. 129.8 MB/s (41159 bytes in 0.000s)
[server] INFO: Device: Xiaomi Redmi Note 4 (Android 7.0)
ERROR: Could not create renderer: Couldn't find matching render driver
scrcpy 1.24 <https://github.com/Genymobile/scrcpy>
/app/share/scrcpy/scrcpy-server: 1 fil...ped. 58.2 MB/s (41159 bytes in 0.001s)
[server] INFO: Device: Xiaomi Redmi Note 4 (Android 7.0)
ERROR: Could not create renderer: Couldn't find matching render driver `
**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

* [x ] Linux
`

   OS: Linux Mint 19.1 Tessa x86_64 
  Host: 80Q7 Lenovo ideapad 300-15ISK 
   Kernel: 4.15.0-101-generic 
  Uptime: 2 hours, 35 mins 
   Packages: 2554 
      Shell: bash 4.4.20 
     Resolution: 1366x768 
   DE: Cinnamon 4.0.10 
  WM: Mutter (Muffin) 
Theme: Mint-Y-Blue (Mint-Y) 
 Theme: Mint-Y [GTK2/3] 
   Icons: Mint-X-Aqua [GTK2/3] 
         Terminal: gnome-terminal 
               CPU: Intel i5-6200U (4) @ 2.800GHz 
                                      GPU: AMD Radeon HD 8670A/8670M/8690M 
                                      GPU: Intel HD Graphics 520 
                                      Memory: 4820MiB / 11868MiB `

**Additional context**

**Comment by viarotel** [comment]
have a try [Escrcpy](https://github.com/viarotel-org/escrcpy)?

**Comment by Komarams** [comment]
> have a try [Escrcpy](https://github.com/viarotel-org/escrcpy)?

interesting, did not know abou that. will try
